joomla/joomla-cms
# [4.0-dev] Add userAgent check

Pull Request for Issue #16172

### Summary of Changes

Set default useragent on all JHttpFactory calls #16172

### Testing Instructions

Do a call with JHttpFactory and confirm that the UserAgent is set.
Set a useragent & try again

### Expected result

userAgent is set.

### Actual result

userAgent is not set.

### Documentation Changes Required

None
